chemprop/args.py

```diff
@@ -546,14 +582,37 @@ def process_args(self) -> None:
                 for key, value in config.items():
                     setattr(self, key, value)
 
+        # Determine the target_columns when training atomic and bond targets
+        self.is_atom_bond_targets = False
+        if self.atom_targets != [] or self.bond_targets != []:
+            self.is_atom_bond_targets = True
+            self.target_columns = self.atom_targets + self.bond_targets
```

Comment:
So we're not enabling molecule-targets alongside atom-bond?

Reply:
My first thought is to let chemprop have the capability to train on atom- and bond-level targets. The predicted atom- and bond-level targets can further be used as additional features in molecule properties prediction. However, It makes sense that people would want to train on atom-level, bond-level, and molecule targets at the same time because it might improve the performance of the model. Do you think it would be better to support this feature in current PR? It might just take a few days to finish this part.

Reply:
Having mixed targets seems like a special case. We can leave it off for now unless you see a convenient way to implement it.

Reply:
Though, I will say, it's possible our current project would benefit from mixed targets.